pegasus/sites.v3/code.org/views/header2.haml

```diff
@@ -40,9 +40,7 @@
 
       #right
         #sign_in_or_user{:style=>'display:block; float: left; margin: .6875em 0 0 30px;'}
-        - hamburger_class = (user_type == "student" || user_type == "teacher" ? "" : "mobile-feature")
-        %span{class: hamburger_class}
-          = view :hamburger, hamburger_options
+        = view :hamburger, hamburger_options
```

Comment:
Yes, the hamburger knows about `user_type` and has its own logic to show/hide the entire hamburger depending on that.  I don't think this code was doing anything any more, and at any rate it shouldn't need to.